ember/app/models/release-version.js

```diff
@@ -1,10 +1,10 @@
 import Model, { attr, belongsTo } from '@ember-data/model';
+import { tracked } from '@glimmer/tracking';
```

Comment:
As mentioned above: move intermediate state to modifying component.

Reply:
im not sure how to then properly implement that with my current form validation

Reply:
Well, then leave it as is for right now.